appeals/api/src/server/endpoints/integrations/integrations.controller.js

```diff
@@ -98,65 +78,43 @@ export const postAppealSubmission = async (req, res) => {
  * @returns {Promise<Response>}
  */
 export const postLpaqSubmission = async (req, res) => {
-	const { caseReference, questionnaire, nearbyCaseReferences, documents } =
-		mapQuestionnaireSubmission(req.body);
-	const dbSavedResult = await importLPAQuestionnaire(
+	const { caseReference, questionnaire, documents } = messageMappers.mapQuestionnaireSubmission(
+		req.body
+	);
+	const dbResult = await integrationService.importLPAQuestionnaire(
+		// @ts-ignore
```

Comment:
What were the type issues that required a ts-ignore?

Reply:
Import commands (appellant case and lpa questionnaire) are not yet defined (besides the December proof of concept). We will be able to work with FO to define them as soon as the appeal schema is finalised.